microk8s-resources/wrappers/microk8s-helm.wrapper

```diff
@@ -1,15 +1,3 @@
 #!/bin/bash
 
-set -eu
-
-export PATH="$SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH"
-ARCH="$($SNAP/bin/uname -m)"
-export LD_LIBRARY_PATH="$SNAP/lib:$SNAP/usr/lib:$SNAP/lib/$ARCH-linux-gnu:$SNAP/usr/lib/$ARCH-linux-gnu"
```

Comment:
I am a bit nervous about dropping this from the classic snap.

Reply:
Dropped due to this, can re-add if we feel safer for possible future changes:
```
$ ldd ./build-scripts/.install/bin/helm 
	not a dynamic executable
```